modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java

```diff
@@ -577,6 +598,27 @@ protected DEDataEnginePermissionSupport getDEDataEnginePermissionSupport() {
 		return new DEDataEnginePermissionSupport(groupLocalService);
 	}
 
+	protected DEDataEngineRequestExecutor getDEDataEngineRequestExecutor() {
```

Comment:
@mclomello Please, remove this hunk (methods getDEDataEngineRequestExecutor and getDeDataRecordCollectionSearchExecutor.

Reply:
Yes Gabriel. Like we talk earlier, I remove and adjust my implementation to work with the new approach.